client/client.go

```diff
@@ -535,6 +540,35 @@ func (c *Client) Collect(ch chan<- prometheus.Metric) {
 	}
 }
 
+// argListFromMgDbParams creates an array of strings from the pointer to the parameters for pt-mongodb-sumamry
+func argListFromMongoDBParams(pParams *agentpb.StartActionRequest_PTMongoDBSummaryParams) []string {
+	var args []string
+
+	// Only adds the arguments are valid
+
+	if pParams.Username != "" {
+		args = append(args, "--username", pParams.Username)
+	}
+
+	if pParams.Password != "" {
+		// TBD change this line when pt-mongodb-summary is updated
```

Comment:
```suggestion
		// TODO change this line when pt-mongodb-summary is updated
```

Reply:
How it should be changed? can we use `--password` here?

Reply:
I mean, we would use it the same way as in for mySQL or postgreSQL, it means two strings to the array: { "--pasword", .Password }. The other PTs accept it without "=", the only exception is for pt-mongodb-summary's password param.
Yes, we can use --password but it will require "=" with this version of pt.